src/work-allocation/components/constants/config.constants.ts

```diff
@@ -34,6 +34,12 @@ const LOCATION: TaskFieldConfig = {
   columnLabel: 'Location',
   views: TaskView.ALL_VIEWS
 };
+const DERIVEDICON: TaskFieldConfig = {
```

Comment:
`DERIVED_ICON` would be a more consistent name.

Reply:
agree with consistency and have made change thanks